### PR TITLE
Added backend build publish on CI

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

**Review by RomanPudashkin — APPROVED** [review]

File: .github/workflows/ci_backend.yml [hunk 1]
```diff
@@ -30,15 +30,9 @@ jobs:
...
-        fi
-
-        DO_BUILD='true'
-
-        if [ $DO_BUILD == 'false' ]; then
-          DO_PUBLISH='false'
-        fi
+        if [[ "${{ github.event.inputs.publish }}" == "on" ]]; then DO_PUBLISH='true'; fi
+        if [ -z "${{ secrets.S3_KEY_CONVERTOR }}" ]; then DO_PUBLISH='false'; fi
+        if [ -z "${{ secrets.S3_SECRET_CONVERTOR  }}" ]; then DO_PUBLISH='false'; fi
```
Comment (Jojo-Schmitz) [inline .github/workflows/ci_backend.yml:35]: Is it really `...CONVERTOR` and not `...CONVERTER`?

File: build/ci/backend/publish.sh [hunk 2]
```diff
@@ -17,4 +17,87 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
\ No newline at end of file
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+S3_KEY=""
+S3_SECRET=""
+S3_URL="s3://convertor.musescore.org"
```
Comment (Jojo-Schmitz) [inline build/ci/backend/publish.sh:24]: here too

File: build/ci/backend/docker/install_mu.sh [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-S3_URL=https://s3.us-east-2.amazonaws.com/convertor.musescore.org
+S3_URL=https://s3.amazonaws.com/convertor.musescore.org
```
Comment (Jojo-Schmitz) [inline build/ci/backend/docker/install_mu.sh:3]: and here

File: build/ci/backend/publish.sh (hunk 2 above)
Comment (igorkorsukov) [reply build/ci/backend/publish.sh:24]: I'll rename these later, it's more difficult with it

File: build/ci/backend/docker/install_mu.sh (hunk 3 above)
Comment (igorkorsukov) [reply build/ci/backend/docker/install_mu.sh:3]: I'll rename these later, it's more difficult with it

**Pull request merged**